docs/ske/01-backstage.mdx

```diff
@@ -172,6 +195,14 @@ apiRouter.use("/ske", await createRouter({ logger, config }));
 Open the `packages/backend/src/plugins/scaffolder.ts` file and add the SKE actions. Your
 `scaffolder.ts` should look something like this:
 
+:::tip
```

Comment:
Perhaps the `tip` should sit above the opening paragraph or below the snippet as the position after the ":" almost interrupts the flow directing you the code snippet

Reply:
done!